runtime_config_linux.go

```diff
@@ -200,20 +200,20 @@ type Network struct {
 type Resources struct {
 	// DisableOOMKiller disables the OOM killer for out of memory conditions
 	DisableOOMKiller bool `json:"disableOOMKiller"`
```

Comment:
isn't the default false?

On Wed, Nov 18, 2015 at 3:22 PM, Mrunal Patel <notifications@github.com>
wrote:

> In runtime_config_linux.go
> <https://github.com/opencontainers/specs/pull/233#discussion_r45276965>:
>
> > @@ -200,20 +200,20 @@ type Network struct {
> >  type Resources struct {
> >  	// DisableOOMKiller disables the OOM killer for out of memory conditions
> >  	DisableOOMKiller bool `json:"disableOOMKiller"`
>
> Should this be a pointer too?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/opencontainers/specs/pull/233/files#r45276965>.
>